### PR TITLE
Add colour to `lf hid` commands

### PR DESCRIPTION
**Pull request by dunderhay**

### CODE REVIEW

File: client/cmdlfhid.c [hunk 1]
```diff
@@ -431,7 +431,7 @@ static int CmdHIDBrute(const char *Cmd) {
                 cmdp++;
                 break;
             default:
-                PrintAndLogEx(WARNING, "Unknown parameter '%c'", param_getchar(Cmd, cmdp));
+                PrintAndLogEx(WARNING, _YELLOW_("Unknown parameter '%c'"), param_getchar(Cmd, cmdp));
```
Comment (iceman1001) [inline client/cmdlfhid.c]: how about just coloring the param part '%c' , not the text.   Makes it easy to spot the important part.

File: client/cmdlfhid.c [hunk 2]
```diff
@@ -400,7 +400,7 @@ static int CmdHIDBrute(const char *Cmd) {
                 param_getstr(Cmd, cmdp + 1, format, sizeof(format));
                 format_idx = HIDFindCardFormat(format);
                 if (format_idx == -1) {
-                    PrintAndLogEx(WARNING, "Unknown format: %s", format);
+                    PrintAndLogEx(WARNING, _YELLOW_("Unknown format: %s"), format);
```
Comment (iceman1001) [inline client/cmdlfhid.c]: how about just coloring the param part '%c' , not the text.   Makes it easy to spot the important part.

File: client/cmdlfhid.c [hunk 3]
```diff
@@ -241,14 +241,14 @@ static int CmdHIDDemod(const char *Cmd) {
...
+            PrintAndLogEx(SUCCESS, "HID Prox TAG (Kastle format) ID: " _GREEN_("%x%08x (%u)")"- Format Len: 32bit - CC: %u - FC: %u - Card: %u", hi, lo, (lo >> 1) & 0xFFFF, cc, fc, cardnum);
         } else {
-            PrintAndLogEx(SUCCESS, "HID Prox TAG ID: %x%08x (%u) - Format Len: %ubit - OEM: %03u - FC: %u - Card: %u",
+            PrintAndLogEx(SUCCESS, "HID Prox TAG ID: " _GREEN_("%x%08x (%u)")"- Format Len: %ubit - OEM: %03u - FC: %u - Card: %u",
                           hi, lo, cardnum, fmtLen, oem, fc, cardnum);
         }
     }
 
-    PrintAndLogEx(DEBUG, "DEBUG: HID idx: %d, Len: %zu, Printing Demod Buffer:", idx, size);
+    PrintAndLogEx(DEBUG, "DEBUG: HID idx: " _GREEN_("%d, Len: %zu")", Printing Demod Buffer:", idx, size);
```
Comment (iceman1001) [inline client/cmdlfhid.c]: I think you took too much here.  Should end at  %d  no?

File: client/cmdlfhid.c [hunk 4]
```diff
@@ -171,17 +171,17 @@ static int CmdHIDDemod(const char *Cmd) {
...
+            PrintAndLogEx(DEBUG, "DEBUG: Error - " _RED_("HID problem during FSK demod"));
         else if (idx == -4)
-            PrintAndLogEx(DEBUG, "DEBUG: Error - HID preamble not found");
+            PrintAndLogEx(DEBUG, "DEBUG: Error - " _RED_("HID preamble not found"));
         else if (idx == -5)
-            PrintAndLogEx(DEBUG, "DEBUG: Error - HID error in Manchester data, size %zu", size);
+            PrintAndLogEx(DEBUG, "DEBUG: Error - " _RED_("HID error in Manchester data, size %zu, size"));
         else
-            PrintAndLogEx(DEBUG, "DEBUG: Error - HID error demoding fsk %d", idx);
+            PrintAndLogEx(DEBUG, "DEBUG: Error - " _RED_("HID error demoding fsk %d, idx"));
```
Comment (iceman1001) [inline client/cmdlfhid.c]: I dont think this will compile..
 _RED_("HID error demoding fsk %d") , idx...

**Comment by dunderhay** [comment]
That was pretty fun to do some commits while you live stream! Really cool thanks for sharing / teaching

**Pull request merged**